src/ontology/mondo-edit.obo

```diff
@@ -110614,7 +110618,9 @@ id: MONDO:0005676
 name: borna disease
 def: "An encephalomyelitis of horses, sheep and cattle caused by borna disease virus." [MESH:D001890]
 comment: This disease is considered zoonotic, i.e. able to be transmitted between human and non-human animals (PMID:34376142). In Mondo, we distinguish between human and non-human animal diseases, and we currently do not have a general term to represent a species-agnostic version of the disease. This record is for the non-human version of “borna disease”.
+subset: nord_rare {source="MONDO:NORD"}
 subset: otar {source="MONDO:OTAR"}
+subset: rare
```

Comment:
Should this be in the `rare` subset? If Yes, then it needs the annotation: `excluded_from_qc_check` and value  `mondoSparqlQcMondo:qc-animal-disease-rare.sparql`

Reply:
No it should not - we need to add the "animal disease in RD QC check" to the Mondo Ingest repo and ensure this error does not come through.

Reply:
The check sparql/qc/mondo/qc-animal-disease-rare.sparql is added in the mondo-ingest `externalclingenmedgenefo` branch.